components/styleguide/typography/paragraph/paragraph.html.twig

```diff
@@ -0,0 +1,9 @@
+{#
+  /**
+   * @file
+   * This file creates some paragraphs of text.
+   */
+#}
+<p>This is a paragraph, <a href="#">and this styled item is a link</a> inside the paragraph. Enjoy a new take on an old favourite with this pear tarte tartin recipe, Aenean lacinia bibendum nulla sed consectetur. Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Vivamus sagittis lacus vel augue laoreet rutrum faucibus.</p>
```

Comment:
What is the difference between the body and the paragraph now? If we add the paragraph, I'd suggest to remove the body.

Reply:
The body has the items for box-sizing, and also a `margin: 0` so there is no space around the edge of the screen.

Reply:
body directory is now deleted from typography directory and `body` css moved back to base.css